catalog.html

```diff
@@ -0,0 +1,291 @@
+              <legend class="filters__title">Стоимость, ₽:</legend>
+              <ul class="filters__list">
+                <li class="filters__item">
+                  <label class="filters__label">от
+                    <input class="filters__input" type="number" name="min-price" value="0">
+                  </label>
+                  <label class="filters__label">до
+                    <input class="filters__input" type="number" name="max-price" value="9000">
+                  </label>
+                </li>
+              </ul>
+            </fieldset>
+            <button class="filters__button" type="submit">Применить</button>
+            <button class="filters__button filters__button--reset" type="reset">Сбросить</button>
+          </form>
+        </section>
+      </div>
+      <section class="main-inner__catalog-hotels catalog-hotels">
+        <h2 class="catalog-hotels__title">Найдено гостиниц: 38</h2>
+        <form class="catalog-hotels__sorting sorting" action="https://echo.htmlacademy.ru/" method="post">
+          <select class="sorting__select" aria-label="Сортировка" name="sorting">
+            <option class="sorting__option" value="cheap"> Сначала дешевые</option>
+            <option class="sorting__option" value="expensivet"> Сначала дорогие</option>
+            <option class="sorting__option" value="rating"> По рейтингу</option>
+          </select>
+        </form>
+        <ul class="catalog-hotels__view view">
+          <li class="view__item">
+            <button class="view__grid" type="button">
+              <span class="visually-hidden">Показать сеткой </span>
+            </button>
+          </li>
+          <li class="view__item">
+            <button class="view__one" type="button">
+              <span class="visually-hidden">Показать по одному </span>
+            </button>
+          </li>
+          <li class="view__item">
+            <button class="view__list" type="button">
+              <span class="visually-hidden">Показать списком </span>
+            </button>
+          </li>
+        </ul>
+        <ul class="catalog-hotels__list-hotels list-hotels">
+          <li class="list-hotels__item">
+            <a class="list-hotels__link" href="#">
+              <img class="list-hotels__img" src="images/catalog/hotel-1.jpg" width="300" height="212"
+                alt="Звездное небо, вид из каньона.">
+            </a>
+            <a class="list-hotels__link" href="#">
+              <h3 class="list-hotels__title">Amara Resort & Spa</h3>
+            </a>
+            <p class="list-hotels__text">Гостиница</p>
+            <p class="list-hotels__cost">От 4000 ₽</p>
+            <a class="list-hotels__detailed" href="#">подробнее</a>
+            <button class="list-hotels__favourites" type="button">в избранное</button>
+            <p class="list-hotels__stars stars stars--stars-four">
+              <span class="visually-hidden"> 4 Звезды</span>
+            </p>
```

Comment:
```suggestion
            <p class="stars stars--4">
              <span class="visually-hidden">4 Звезды</span>
            </p>
```


Я бы не стала просить программиста прикручивать словарь соответствия числа и слова. В базе данных количество звезд будет числом. Отправить его же в класс очень просто. А для конвертации в текст нужна таблица соответствия. А смысла в том, что класс будет словом, а не числом нет.